easypost/report.py

```diff
@@ -7,7 +7,20 @@ class Report(AllResource, CreateResource):
     @classmethod
     def create(cls, api_key=None, **params):
         requestor = Requestor(api_key)
-        url = "%s/%s" % (cls.class_url(), params["type"])
+        url = "%s/%s" % (cls.class_url(), params["type"]) + "?"
+
+        if "columns" in params:
+            for column in params.get("columns", []):
+                url += "columns[]=" + str(column) + "&"
```

Comment:
Nit: Since we only support 3.6+ now, we can start using f-strings

Reply:
good suggestion